### ISSUE TITLE
verbose params for functions

### ISSUE THREAD

**Issue by brianleroux** [architect/architect#1014]
with functions open to extension propose we allow `arc.config` to go inline. for example:

```
@app
myapp

@http
/
  method get
  src /path/to/app
  runtime python3.8
  memory 1152
  timeout 30
  concurrency 1
```

we'd have to decide how to handle if `config.arc` is present; probably fail and force them to choose.

**Comment by ryanblock** [comment]
I considered this! Ultimately for the first go I figured, based on conversations that we'd had, that the project manifest would be for project level config. (That in addition the fact that function level config could really bloat it up.) Thoughts?

**Comment by brianleroux** [comment]
yeah I'm not itching for this just think its a logical extension; not an either/or but rather an option. don't like local config? define it globally. need to override? define it verbosely OR add `config.arc` for that function. just options.

**Comment by filmaj** [comment]
i never realized the discrepancy in having both a app.arc and .arc-config file before, but looking at it now, i'm in favour of this. so the .arc-config options become options that are 'global' to all functions, regardless of pragma / infra type, whereas the current `@http` options become http-only ones. i think it's easier to communicate what you can configure at a function infra level by putting everything in one place, so i think this is a docs improvement too.

**Comment by ryanblock** [comment]
@filmaj fwiw you can put it all in once place today:
